solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -607,19 +609,31 @@ public PackageStoreAPI getPackageStoreAPI() {
   public void load() {
     log.debug("Loading cores into CoreContainer [instanceDir={}]", loader.getInstancePath());
 
+    // Always add $SOLR_HOME/lib to the shared resource loader
+    Set<String> libDirs = new HashSet<>();
+    libDirs.add("lib");
+
+    if (!StringUtils.isBlank(cfg.getSharedLibDirectory())) {
+      List<String> sharedLibs = Arrays.asList(cfg.getSharedLibDirectory().split("\\s*,\\s*"));
```

Comment:
I'm almost certain we've got a utility that does this so that you don't have to define your regexp

Reply:
I tried `StrUtils.splitSmart()` but we don't need the smartness of not splitting inside quotes etc, and it does not trim whitespace around comma. So I'll keep this unless you know about a better split-util somewhere.